harvard-university-for-the-creative-arts.csl

```diff
@@ -290,6 +276,7 @@
           <et-al font-style="italic"/>
         </names>
         <text variable="medium" prefix=" [" suffix="] "/>
+        <text variable="genre" prefix=" [" suffix="]"/>
```

Comment:
The subsequent square brackets seem wrong -- shouldn't those be in a group (or have a test for one or the other between medium and genre)?

Reply:
Looking at my test references I don't think that I need genre, there as for all sources that use Motion Picture the medium will be used to describe what the source is. I can take genre out, will that fix the problem?